src/otx/algo/segmentation/model/utils/__init__.py

```diff
@@ -0,0 +1,13 @@
+"""Utils used for mmseg model."""
```

Comment:
Let's unify the format of the license --> docstring format, for a while.

Reply:
Same here -> https://github.com/openvinotoolkit/training_extensions/pull/2693/files#diff-fff119407ff45ba5eca03ce184fec51f68e89857803b4b7c46473f94e7fee345R1